src/libs/IOUUtils.ts

```diff
@@ -53,19 +53,39 @@ function navigateToParticipantPage(iouType: ValueOf<typeof CONST.IOU.TYPE>, tran
 }
 
 /**
- * Calculates the amount per user given a list of participants
+ * Calculates the amount per split.
  *
- * @param numberOfParticipants - Number of participants in the chat. It should not include the current user.
+ * @param numberOfSplits - Number of splits EXCLUDING the remainder holder ("default user").
  * @param total - IOU total amount in backend format (cents, no matter the currency)
- * @param currency - This is used to know how many decimal places are valid to use when splitting the total
- * @param isDefaultUser - Whether we are calculating the amount for the current user
+ * @param currency - Used to know how many decimal places are valid when splitting the total
+ * @param isDefaultUser - Whether we are calculating the amount for the remainder holder
+ * @param roundingMode - 'round' (default, legacy behavior) or 'floorToLast' to floor all and put full remainder on the default user
  */
-function calculateAmount(numberOfParticipants: number, total: number, currency: string, isDefaultUser = false): number {
+function calculateAmount(numberOfSplits: number, total: number, currency: string, isDefaultUser = false, roundingMode: 'round' | 'floorToLast' = 'round'): number {
```

Comment:
NAB: Since you only have two rounding modes right now this could be a bool instead (`useFloorToLastRounding`) instead of passing a string rounding method.